client/reader/following-stream/index.jsx

```diff
@@ -405,7 +405,7 @@ module.exports = React.createClass( {
 			hasNoPosts = store.isLastPage() && ( ( ! this.state.posts ) || this.state.posts.length === 0 ),
 			body;
 
-		if ( hasNoPosts ) {
+		if ( hasNoPosts || store.hasRecentError() ) {
```

Comment:
This works, but I'm concerned it might be a bit general and might catch other types of errors that aren't related to a tag not being found. Perhaps `hasRecentError` should accept an `errorType` argument?